specification/network/resource-manager/Microsoft.Network/stable/2022-11-01/virtualWan.json

```diff
@@ -8894,6 +9211,61 @@
         }
       }
     },
+    "RoutingConfigurationNfvSubResource": {
+      "type": "object",
+      "properties": {
+        "ResourceUri": {
+          "type": "string",
+          "description": "Resource ID."
+        }
+      },
+      "description": "Reference to RouteTableV3 associated with the connection."
+    },
+    "RoutingConfigurationNfv": {
```

Comment:
what's the difference between this and RoutingConfiguration

Reply:
The difference is in "RoutingConfiguration", for the AssociatedRouteTable, PropagatedRouteTable and the RouteMaps  the reference "#/definitions/SubResource" is used which has resource uri property named as "id" but in the NfvRP version of RoutingConfiguration the resource uri property is named "ResourceUri". Hence I had to define another version of the RoutingConfiguration to match what is there in NFV repo. NVA connection is an NfvRP resource so we have to use the RoutingConfiguration class as defined in NfvRP.